scripts/health.sh

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env bash
+# Check the health status of the current environment
+# If this script executed successfully, ware all set to run.
+set -u
+
+MAJOR_MINIMUM_VERSION_DOCKER="17"
+MINOR_MINIMUM_VERSION_DOCKER="05"
+
+help() {
+  if [ "$1" == "docker" ]; then
+    echo "Please follow the installation instructions on https://docs.docker.com/get-docker/"
+    echo "or for a quick installation for non production linux machines the instruction on https://github.com/docker/docker-install"
+  else
+    echo "There is no suggestion for the issue at hand, please contact your administrator or support"
+  fi
+}
+
+is_command_available () {
+  command -v "$1" > /dev/null
+}
+
+is_version_compatible() {
+  # $1 - expected major version
+  # $2 - expected minor version
+  # $3 - actual major version
+  # $4 - actual minor version
+  if [ "$3" -lt "$1" ]; then
+    return 1
+  fi
+  if [ "$3" -eq "$1" ] && [ "$4" -ge "$2" ]; then
```

Comment:
Hm, is this correct?
Shouldn't be
```suggestion
  if [ "$3" -eq "$1" ] && [ "$4" -lt "$2" ]; then
```

Reply:
I think the current version is correct, but even the function is small it is obviously to hard to read.
I tried to make it more clear by introducing named local variables. 

comments are added only here in the comment snippet.
(hopefully the code is clear enough with the named vars, if not please let me know)
```bash 
  if [ "$major" -lt "$min_major" ]; then
    return 1
  fi
  # if major version was smaller than the required minimum we exited func
  
  # if the major version is equal, we need to make sure the minor version is >= minimum minor
  if [ "$major" -eq "$min_major" ] && [ "$minor" -ge "$min_minor" ]; then
    return 1
  fi
  # if the major version is > we don't care about the minor version

```

Reply:
Or did I miss something?

Reply:
@tomuben as discussed you have been right, the check should be turned around.
I will add the change to the latest version. Thx!